### ISSUE TITLE
Reorganize docs slightly

### ISSUE THREAD

**Issue by kyleabeauchamp** [msmbuilder/msmbuilder-legacy#272]
Re #271 

So I think the current "main" tutorial is pretty good.  

Maybe for the second tutorial, we should _just_ do Ward / alternative clustering / RMSD choices.

Then, we could have a section of "Additional Tutorials" that might include tICA, BACE, SCRE

Obviously the choice of which tutorials belong in the "main" section and which ones are "additional" is up for discussion.  In particular, tICA might very well be a main tutorial.

To summarize, here's my current thinking:
## Tutorials
1.  Main workflow (Cluster, Assign, analyze)
2. Alternative Clustering
3.  Alternative analysis methods (TPT)
## Additional Tutorials (somewhat but not too much hidden)
1.  tICA
2.  BACE
3.  SCRE
